src/components/HeroBanner/utils.js

```diff
@@ -0,0 +1,14 @@
+const helpers = {
+  accessLengthInMonths(accessLength) {
+    let monthsLength = '';
+    if (accessLength !== undefined && accessLength > 0) {
+      const accessLengthMonths = Math.round(accessLength / 30);
```

Comment:
I think we should use `parseInt(accessLength / 30)` instead of `Math.round()` such that we do the same rounding as LMS:
https://github.com/edx/edx-platform/blob/master/openedx/features/journals/templates/journals/journal_card.html#L33-L42